Assets/Mirror/Editor/MirrorAssetProcessor.cs

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Is license really needed here?  we already have MIT license file at the root of the project so it already applies

Reply:
Removed